### PR TITLE
[Deep link] update fingerprint UI

### PR DESCRIPTION
**Pull request by hangyujin**
1. Show different messages when there is a fingerprint in PDC or not
2. Some UI updates

*List which issues are fixed by this PR.*

*Please add a note to `packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md` if your change requires release notes. Otherwise, add the 'release-notes-not-required' label to the PR.*

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or there is a reason for not adding tests.

![build.yaml badge]

If you need help, consider asking for help on [Discord].

[Contributor Guide]: https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[build.yaml badge]: https://github.com/flutter/devtools/actions/workflows/build.yaml/badge.svg

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart [hunk 1]
```diff
@@ -44,8 +47,15 @@ class GenerateAssetLinksResult {
   String generatedString;
 }
 
+class ValidateAndroidDomainResult {
+  ValidateAndroidDomainResult(
+      this.googlePlayFingerprintsAvailability, this.domainErrors);
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart:52]: nit: trailing comma

File: packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart [hunk 2]
```diff
@@ -180,15 +180,77 @@ class _DomainCheckTable extends StatelessWidget {
...
+              padding: const EdgeInsets.all(largeSpacing),
+              child: RoundedOutlinedBorder(
+                child: Padding(
+                  padding: const EdgeInsets.all(largeSpacing),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      if (!hasPdcFingerpint && !haslocalFingerpint) ...[
+                        const Text(
+                          'Fix guide:',
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart:229]: nit: if a Text widget fits on one line, no need for trailing comma. We do this for Text widgets and SizedBox. This helps make the build methods more terse

File: packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart [hunk 3]
```diff
@@ -659,3 +711,23 @@ class _NoIssueText extends StatelessWidget {
...
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Icon(
+          Icons.error,
+          color: Theme.of(context).colorScheme.error,
+          size: defaultIconSize,
+        ),
+        const SizedBox(width: denseSpacing),
+        Text(text),
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart:729]: does this need to be in an Expanded widget?

**Review by kenzieschmoll — APPROVED** [review]
a few comments but lgtm

**Comment by hangyujin** [comment]
This pr is failing benchmark size test, is it ok if i increase the gzipBundleSizeBenchmark from 1400 to 1450?

**Pull request merged**